### DISCUSSION TITLE
Organizations creations for Enterprises

### DISCUSSION THREAD

**Discussion by joerg** [community/community#32018] [Enterprise] [answered]
Hi,

We are moving our Enterprise to Github Cloud and thus we need to create several Organizations. Is there any way to do that in an automated way? I could not find any API endpoint for that and creating them manually for hundreds of organizations is way too error-prone.

regards,
    Jörg

**Comment by byrneh** [comment, marked as answer]
@joerg try the GraphQL API [#createenterpriseorganization](https://docs.github.com/en/enterprise-cloud@latest/graphql/reference/mutations#createenterpriseorganization)
